src/OrbitGl/TrackManager.cpp

```diff
@@ -403,3 +403,31 @@ FrameTrack* TrackManager::GetOrCreateFrameTrack(uint64_t function_id,
 
   return track.get();
 }
+
+uint32_t TrackManager::GetNumTimers() const {
+  uint32_t num_timers = 0;
+  std::lock_guard<std::recursive_mutex> lock(mutex_);
+  for (const auto& track : GetAllTracks()) {
+    num_timers += track->GetNumTimers();
+  }
+  // Frame tracks are removable by users and cannot simply be thrown into the
+  // tracks_ vector.
+  for (const auto& track : GetFrameTracks()) {
+    num_timers += track->GetNumTimers();
+  }
+  return num_timers;
+}
+
+std::pair<uint64_t, uint64_t> TrackManager::GetTracksMinMaxTime() const {
```

Comment:
What about calling this `GetTracksMinMaxTimestamps` for consistency with `UpdateCaptureMinMaxTimestamps` and in general because "timestamp" is more specific than "time"?

Reply:
Of course! Changed.